jupyter-config/rucio_jupyterlab.json

```diff
@@ -1,6 +1,6 @@
 {
-  "NotebookApp": {
-    "nbserver_extensions": {
+  "ServerApp": {
+    "jpserver_extensions": {
```

Comment:
It's normal that the script changes from a notebook server extension to a jupyter server extension. But I don't see in your changes the upgrade. Did you check if it works as is? But, even if that is the case, can we have the migration of the server as well? https://jupyter-server.readthedocs.io/en/latest/developers/extensions.html#migrating-an-extension-to-use-jupyter-server

Reply:
I've pushed the changes, please check :)